packages/jest-serializer/src/__tests__/index.test.ts

```diff
@@ -55,18 +70,28 @@ afterEach(() => {
 v8s.forEach((mockV8, i) => {
   describe('Using V8 implementation ' + i, () => {
     beforeEach(() => {
-      v8.serialize = mockV8.serialize;
-      v8.deserialize = mockV8.deserialize;
+      v8['serialize'] = mockV8.serialize;
+      v8['deserialize'] = mockV8.deserialize;
     });
 
-    it('throws the error with an invalid serialization', () => {
+    it('reject promise when invalid serialization', async () => {
       // No chance this is a valid serialization, neither in JSON nor V8.
       const invalidBuffer = Buffer.from([0, 85, 170, 255]);
 
-      fs.writeFileSync(file, invalidBuffer);
+      await new Promise(resolve =>
+        fs.writeFile(mockFile, invalidBuffer, resolve),
+      );
 
       expect(() => serializer.deserialize(invalidBuffer)).toThrow();
-      expect(() => serializer.readFileSync(file)).toThrow();
+      expect(serializer.readFile(mockFile)).rejects.toThrow();
+    });
+
+    it('rejects promise when file does not exist', () => {
+      expect(serializer.readFile('does-not-exist')).rejects.toThrow();
```

Comment:
async assertions must be `return`ed (or `await`ed) so we wait for them before completing the test

Reply:
Good catch, thanks. Planning on just adding an `async` to the test function and then an `await` on each of the `expect` calls. That should work, yeah?

Reply:
yup, that works